### DISCUSSION TITLE
FluxNews: Audio Player (support background playing)

### DISCUSSION THREAD

**Discussion by minhcasi** [inspireui/support#5079] [Working on Features]
Support Audio Play on FluxNews (Blogs app)

**Comment by kodzodeku** [comment]
This would be an amazing feature that I can even use in my e-learning app store made with Fluxstore app.

**Reply by EmanuelFromFlorence** [reply]
me too , to create a wellness app with pro sections for different area of wellness , single audio player.single item , playlist with monthly subscriptions for the pro sections of the app or for the podcasts.

**Reply by DavideM12** [reply]
Yes, it would be amazing!

**Comment by DavideM12** [comment]
When it will be added?

**Reply by minhcasi** [reply]
This feature is already finished and in the testing process - thank you!

**Comment by EmanuelFromFlorence** [comment]
Really ??
Audio player are coming ?
Single player or player that can have playlist ?


Wonderful
ᐧ

Il giorno mer 21 lug 2021 alle ore 10:23 Minh Pham ***@***.***>
ha scritto:

**Comment by EmanuelFromFlorence** [comment]
Just bought...
just a question
do i need to buy an extended license in order to use subscriptions for
selling my podcast, lessons, audio tracks and playlists?

>  New Subscription Plans Sign Up feature (*Woocommerce or WordPress*
> <https://tppr.me/qpThu>* extended license*), compatible with Ultimate
> Membership Pro <http://1.envato.market/YgqroR>.


I also bought Ultimate Membership pro..

don't tell me my license is not enough

ps.
i can't find in FLuxnews the audio player anywhere...
can you show me how?

thanks
ᐧ

Il giorno mer 21 lug 2021 alle ore 10:59 Manu ***@***.***> ha
scritto:
